src/utils.rs

```diff
@@ -64,3 +70,30 @@ pub fn get_human_readable_timestamp(duration: Duration) -> String {
 pub fn get_full_username(user: &User) -> String {
     format!("{}#{:04}", user.name, user.discriminator)
 }
+
+pub fn get_prefixes() -> serde_json::Value {
```

Comment:
I suggest [this](https://github.com/aquelemiguel/parrot/blob/jc/prefix-change-proposal/src/utils.rs#L74)